fdbbackup/backup.actor.cpp

```diff
@@ -286,6 +316,12 @@ CSimpleOpt::SOption g_rgBackupStartOptions[] = {
 	{ OPT_ENCRYPTION_KEY_FILE, "--encryption-key-file", SO_REQ_SEP },
 	{ OPT_ENCRYPT_FILES, "--encrypt-files", SO_REQ_SEP },
 	{ OPT_BLOB_MANIFEST_URL, "--blob-manifest-url", SO_REQ_SEP },
+
```

Comment:
-v and the other added by you are common (not command-specific) options. Please exclude them from each opti